component/modal/modal.css

```diff
@@ -0,0 +1,39 @@
+.modal{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+
+    width: 60%;
+    background-color: var(--MODAL-BACKGROUND-COLOR);
+    padding: 2rem;
```

Comment:
nice that you structured the code as sudesh mentioned, I have to do it as well :sweat_smile:

Reply:
Making a habit of following this.